docs/src/markdown/index.md

```diff
@@ -12,7 +12,7 @@ RegReplace is a plugin for Sublime Text that allows the creating of commands con
 - Chain multiple regular expression find and replace rules together.
 - Create rules that can filter regular expression results by syntax scope.
 - Create rules that target specific scopes and apply regular expression to the content.
-- Create commands that highlight results and requiring confirmation before replacing.
-- Create find and fold/unfold commands to fold or unfold the find results instead of replacing them.
-- Create find and mark/unmark commands to simply highlight or un-highlight results instead of replacing them.
+- Create commands that highlight results and require confirmation before replacing.
+- Create find and fold/unfold commands to fold or unfold the found results instead of replacing them.
+- Create find and mark/unmark commands to simply highlight or unhighlight results instead of replacing them.
```

Comment:
Just add `unhighlight` to `.dictionary`.

Reply:
Done.